docs/modules/language-reference/pages/index.adoc

```diff
@@ -3179,6 +3179,33 @@ This section discusses language features that are generally more relevant to tem
 <<blank-identifiers,Blank Identifiers>> +
 <<projects,Projects>>
 
+[[the-meaning-of-new]]
+=== The Meaning of `new`
+
+Pkl has no "true" object instantiation: the `new` keyword is a special case of <<amending-objects, amending>>.
```

Comment:
nit: you could rephrase this sentence to be a bit less negative. Something like "Object instantiation in Pkl is actually just amending" or something of the sort.

Reply:
Ditto--don't know what true instantiation is!